src/main/java/com/rescuebites/api/users/services/implementations/UserServiceImpl.java

```diff
@@ -94,7 +94,7 @@ public void resendConfirmationEmail(String email) {
         // Generamos nuevo token y enviamos email
         Token token = tokenService.saveUserToken(user);
         UUID newToken = token.getTokenId();
-        String resendConfirmAccountHtml = emailBuilder.buildResendConfirmAccount(user.getEmail(), newToken);
+        String resendConfirmAccountHtml = emailBuilder.buildResendConfirmAccount(user, newToken);
         emailService.sendEmail(user.getEmail(), "Confirm your registration ✔", resendConfirmAccountHtml);
```

Comment:
Acá, en vez de tener un método tan genérico en el emailService `sendEmail()`, lo que podrías hacer es que el `EmailService` ya conozca los diferentes emails a conocer, y el `UserService` lo único que haría es decirle al `EmailService` qué tipo de correo enviar, algo así:


```suggestion
emailService.sendConfirmAccountEmail(user.getEmail(), newToken);
```

Y dentro de ese método, el emailService solo deberia recibir el correo donde debe enviar el correo y la información dinámica (el nuevo token)